acceptance/bundle/sync-paths-dotdot/script

```diff
@@ -0,0 +1 @@
+$CLI bundle validate
```

Comment:
Oh, I didn't realise it's reproducible by validate as well, struggled to right it for deploy